src/backend/api/Fusion.Resources.Logic/Errors/InvalidWorkflowError.cs

```diff
@@ -0,0 +1,66 @@
+﻿using FluentValidation;
+using FluentValidation.Results;
+using Fusion.Resources.Logic.Workflows;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Fusion.Resources.Logic
+{
+    public class InvalidWorkflowError : ValidationException
+    {
+        //public List<WorkflowFieldError> FieldErrors { get; } = new List<WorkflowFieldError>();
+
+        public string WorkflowTypeName { get; set; }
```

Comment:
Nullable ?

Reply:
Did that in the change request branch :)